cypress/e2e/moderationDashboard.cy.ts

```diff
@@ -116,25 +145,54 @@ describe('Moderation Facility Submission Form', () => {
             ]
         }
     }
-    context('Landscape mode', () => {
-        beforeEach(() => {
-            // The resolution is in the beforeEach() instead of before() to
+        before(() => {
+                           // The resolution is in the beforeEach() instead of before() to
             // prevent Cypress from defaulting to other screen sizes between tests.
-            cy.viewport(1920, 1080)
-            cy.visit('/moderation')
-            cy.intercept('POST', findADocJapanAPIEndpoint, req => {
-                if (req.body.query && req.body.query.includes('query Submissions')) {
-                    req.reply({
-                        statusCode: 200,
-                        body: mockedSubmissionResponse
-                    })
-                }
-            }).as('getSubmissions')
-            cy.wait('@getSubmissions')
-            // This wait time is to give the page elements time to load.
-            cy.wait(2000)
-            cy.get('[data-testid="mod-submission-list-item-1"]').click()
-            cy.wait(2000)
+             cy.viewport(1920, 1080)
+
+                
+
+                cy.visit('/login')      
+                Cypress.session.clearCurrentSessionData() 
+
+                // Wait for redirect to login form
+                cy.wait(3000)
+
+                cy.origin('https://findadoc.jp.auth0.com/', () => {
+                    cy.get('input#username').should('be.visible').type('findadoctest@proton.me')
+                    cy.get('[data-action-button-primary]').should('be.visible').click()
+                    cy.get('input#password').should('be.visible').type('vCnL5J8agHg6m2f')
+                    cy.get('[data-action-button-primary]').should('be.visible').click()
+                })
+                cy.wait(3000)
+                
+                cy.intercept('POST', '**/', (req) => {
+                    req.continue((res) => {
+                        console.log(req.body.query);
```

Comment:
Non-blocking: Do we need this console log here?

Reply:
I actually think it's helpful to see the queries we've intercepted in tests, but technically we don't need it